docs/rst/statistics_backend/get_data.rst

```diff
@@ -0,0 +1,118 @@
+| |ACKNACK_COUNT-api|           | |DATAREADER-api|                   | Not applicable                     |
++-------------------------------+------------------------------------+------------------------------------+
+| |NACKFRAG_COUNT-api|          | |DATAREADER-api|                   | Not applicable                     |
++-------------------------------+------------------------------------+------------------------------------+
+| |GAP_COUNT-api|               | |DATAWRITER-api|                   | Not applicable                     |
++-------------------------------+------------------------------------+------------------------------------+
+| |DATA_COUNT-api|              | |DATAWRITER-api|                   | Not applicable                     |
++-------------------------------+------------------------------------+------------------------------------+
+| |PDP_PACKETS-api|             | |PARTICIPANT-api|                  | Not applicable                     |
++-------------------------------+------------------------------------+------------------------------------+
+| |EDP_PACKETS-api|             | |PARTICIPANT-api|                  | Not applicable                     |
++-------------------------------+------------------------------------+------------------------------------+
+| |DISCOVERED_ENTITY-api|       | |PARTICIPANT-api|                  | Not applicable                     |
++-------------------------------+------------------------------------+------------------------------------+
+| |SAMPLE_DATAS-api|            | |DATAWRITER-api|                   | Not applicable                     |
++-------------------------------+------------------------------------+------------------------------------+
+
+
+
+.. _statistics_backend_get_data_examples:
+
+Examples
+^^^^^^^^
+
+Following, some example queries are provided to serve a inspiration for applications using
+*Fast DDS Statistics Backend*.
+
+DataWriter's Fast DDS Latency median example
+""""""""""""""""""""""""""""""""""""""""""""
+
+.. literalinclude:: /code/StatisticsBackendTests.cpp
+   :language: c++
+   :start-after: //CONF-GET-DATA-DATAWRITER-FASTDDS_LATENCY
+   :end-before: //!
+   :dedent: 8
+
+Topic's Fast DDS Latency mean example
+"""""""""""""""""""""""""""""""""""""
+
+.. literalinclude:: /code/StatisticsBackendTests.cpp
+   :language: c++
+   :start-after: //CONF-GET-DATA-TOPIC-FASTDDS_LATENCY
+   :end-before: //!
+   :dedent: 8
+
+Topic's Heartbeat count maximum example
+"""""""""""""""""""""""""""""""""""""""
+
+.. literalinclude:: /code/StatisticsBackendTests.cpp
+   :language: c++
+   :start-after: //CONF-GET-DATA-TOPIC-HEARTBEAT_COUNT
+   :end-before: //!
+   :dedent: 8
+
+Host to Host Fast DDS Latency all points example
+""""""""""""""""""""""""""""""""""""""""""""""""
+
+It is also possible to retrieve all the data points of a given |DataKind-api| within the time frame.
```

Comment:
I will add this under a different title not related with and example, and then the example.
Maybe move it to the head of the page.

Reply:
I've instead added a sentence explaining what happens with bins is 0